features/hooks_failures_json.feature

```diff
@@ -0,0 +1,854 @@
+Feature: Display hook failures location in json printer
+  In order to be able to locate the code that generated a failure
+  As a feature developer using the json printer
+  When a hook throws an error I want to see the related item where the code failed
+
+  Background:
+    Given a file named "features/one.feature" with:
```

Comment:
Out of curiosity, why inline these feature & context files rather than creating a fixture directory?

Reply:
I believe that this test can probably share the fixtures with the other tests for the hook failures for the other formatters. But that would require changing those tests to use fixtures, so I would prefer to do that as a separate PR